app/uk/gov/hmrc/debttransformationstub/controllers/TimeToPayController.scala

```diff
@@ -257,3 +269,20 @@ class TimeToPayController @Inject() (
     headers.get("correlationId").getOrElse(throw new Exception("Missing required correlationId header"))
 
 }
```

Comment:
Putting the method above this curly brace will resolve some of the compilation errors